code/modules/hydroponics/grown.dm

```diff
@@ -85,6 +85,49 @@
 			for(var/datum/plant_gene/trait/T in seed.genes)
 				T.on_attackby(src, O, user)
 
+//Ghetto Seed Extraction
+/obj/item/reagent_containers/food/snacks/grown/attackby(obj/item/W, mob/user, params)
+	switch(W)
+		if(W.get_sharpness())
+			playsound(loc, 'sound/weapons/slice.ogg', 50, TRUE, -1)
+			user.visible_message("<span class='notice'>[user] starts slicing apart \the [src].</span>", "<span class='notice'>You start slicing apart \the [src]...</span>", "<span class='hear'>You hear the sound of a sharp object slicing some plant matter.</span>")
+			if(do_after(user, 50, target = src))
+				to_chat(user, "<span class='notice'>You slice apart the [src]! You went too far and the tiny remaining scraps are worthless!</span>")
+			/*	new /obj/item/reagent_containers/food/snacks/grown/sliced(user.drop_location(), 1)
+				use(1) */
+				seedify(src, 1)
```

Comment:
```suggestion
	if(W.get_sharpness())
		playsound(loc, 'sound/weapons/slice.ogg', 50, TRUE, -1)
		user.visible_message("<span class='notice'>[user] starts slicing apart \the [src].</span>", "<span class='notice'>You start slicing apart \the [src]...</span>", "<span class='hear'>You hear the sound of a sharp object slicing some plant matter.</span>")
		if(do_after(user, 50, target = src))
			to_chat(user, "<span class='notice'>You slice apart the [src]! You went too far and the tiny remaining scraps are worthless!</span>")
		/*	new /obj/item/reagent_containers/food/snacks/grown/sliced(user.drop_location(), 1)
			use(1) */
			seedify(src, 1)
	switch(W.tool_behaviour)

Reply:
Removing the faulty ``switch()`` and putting the right one at the bottom, also fixing indentation.